src/actions/guides/database/models.cr

```diff
@@ -7,70 +8,160 @@ class Guides::Database::Models < GuideAction
+
+    Models in Lucky allow you to define methods associated with each column in the table. These methods return the value set in that column.
+
+    ## Generate a model
 
-    Let's create a `User` model with `lucky gen.model User` and then add some
-    columns. The name is a `String` and the age is an `Int32?`. The `age` column
-    allows null because the type is nilable (designated by the `?`).
+    Lucky gives you a task for generating a model along with several other files that you will need for interacting with your database.
+
+    Use the `lucky gen.model {ModelName}` task to generate your model. If you're generating a `User` model, you would run `lucky gen.model User`. Running this will generate a few files for you.
+
+    * [User model](##{ANCHOR_SETTING_UP_A_MODEL}) - Located in `./src/models/user.cr`
+    * [User form](#{Guides::Database::ValidatingSavingDeleting.path}) - Located in `./src/forms/user_form.cr`
+    * [User query](#{Guides::Database::Querying.path}) - Located in `./src/queries/user_query.cr`
+    * [User migration](#{Guides::Database::ManagingAndMigrating.path}) - Location in `./db/migrations/#{Time.utc.to_s("%Y%m%d%H%I%S")}_create_users.cr`
+
+    #{permalink(ANCHOR_SETTING_UP_A_MODEL)}
+    ## Setting up a model
 
-    > Note: we'll assume we ran a migration that created a name and age column.
+    Once you run the model generator, you'll have a file that looks like this
 
     ```crystal
     # src/models/user.cr
     class User < BaseModel
       table :users do
-        column name : String
-        column age : Int32?
+        # You will define columns here
       end
     end
     ```
 
-    > You can add a `?` to the end of any type to mark it as nilable.
+    Your model will inherit from `BaseModel` which is just an abstract class. You can use this to define methods all of your models should have access to.
+    Next you'll see the `table` block that defines which table this model is connected to. If your table defined the primary key column as `UUID`, you'll need to add the `primary_key_type` option here as well.
+
+    ```crystal
+    # src/models/user.cr
+    class User < BaseModel
+      # Sets the type for `id` to `UUID`
+      table :users, primary_key_type: :uuid do
+      end
+    end
+    ```
+
+    ## Defining a column
+
+    ### Default columns
+
+    By default, Lucky will add a few columns to your model for you.
+
+    * `id` - Your primary key column. Default `Int32` unless you specified `UUID`.
+    * `created_at` - default `Time` type.
+    * `updated_at` - default `Time` type.
+
+    > These columns are autogenerated and managed by Avram for you.
+
+    ### Adding a column
+
+    Inside of the `table` block, you'll add the columns your model will define using the `column` method.
+
+    ```crystal
+    table :users do
+      column email : String
+      column active : Bool
+      column ip_address : String?
```

Comment:
I think it'd be valuable to add a comment here saying `This type can be nil because it ends with '?'`. Even though it's mentioned earlier I think having it here would help